packages/pwa/app/pages/account/index.jsx

```diff
@@ -74,7 +74,8 @@ const Account = () => {
                     <SignoutIcon boxSize={5} mr={2} />
                     <Text as="span" fontSize={['md', 'md', 'md', 'sm']} fontWeight="normal">
                         {formatMessage({
-                            defaultMessage: 'Log out'
+                            defaultMessage: 'Log out',
+                            id: 'account.logout_button.button.log_out'
```

Comment:
What do you think of this id? 
```suggestion
                            id: 'account_page.button.log_out'
```

Reply:
I think I prefer to have a more specific hierarchy in this case: Account page → LogoutButton component

Reply:
oh I see, sorry I did not see there is a LogOut button there